apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -108,33 +103,33 @@ export const useTableColumns = (
             {
                 title: NumberMessage,
                 sortOrder: get(sorterMap, 'number'),
-                filteredValue: getFilteredValue<IFilters>(filters, 'number'),
+                filteredValue: getFilteredValue(filters, 'number'),
```

Comment:
more types provided better then less types. is removing types here is essential?

Reply:
return generic on all `getFilteredValue` here